pkg/cloudprovider/aws/aws.go

```diff
@@ -22,18 +22,26 @@ import (
 	"net"
 	"regexp"
 	"strings"
+	"sync"
 
 	"code.google.com/p/gcfg"
 	"github.com/mitchellh/goamz/aws"
 	"github.com/mitchellh/goamz/ec2"
+	"github.com/mitchellh/goamz/elb"
 
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/resource"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/cloudprovider"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
 
 	"github.com/golang/glog"
 )
 
+const LOADBALANCER_NAME_PREFIX = "k8s-"
+const LOADBALANCER_TAG_NAME = "k8s:name"
+const LOADBALANCER_NAME_MAXLEN = 32 // AWS limits load balancer names to 32 characters
+
+// TODO: Should we rename this to AWS (EBS & ELB are not technically part of EC2)
```

Comment:
I think either is fine, but it'd be help for mentally keeping things straight if this was named the same as the directory and file it's in. It'd be best in a separate PR though, not here, if you want to clean it up.